example/server.proteus.go

```diff
@@ -2,6 +2,7 @@ package example
 
 import (
 	"golang.org/x/net/context"
+	"gopkg.in/src-d/proteus.v1/example/categories"
```

Comment:
This file is a generated file, is it really important for a generated file to follow that rule?